### PR TITLE
#159082303 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### What does this PR do?
Add Continous Integration and README badge using Travis CI and Coverage
#### What are the relevant pivotal tracker stories?
#159082303

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - DJANGO_VERSION=2.0.7
+
+before_script:
+  - psql -c 'create database test_db;' -U postgres
+ 
+install:
+  - pip install -r requirements.txt
+
+script:
+  - coverage run --source='.' manage.py test
```
Comment (pluwum) [inline .travis.yml]: Do we have coverage installed?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  - psql -c 'create database test_db;' -U postgres
+ 
+install:
+  - pip install -r requirements.txt
+
+script:
+  - coverage run --source='.' manage.py test 
+
+after_success:
+  - coveralls
```
Comment (pluwum) [inline .travis.yml:29]: Do we have coveralls installed?

File: .coveragerc [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[report]
+show_missing = True
+omit =
+    *__init__*
```
Comment (pluwum) [inline .coveragerc]: Just add new lines after this statement

**Review by pluwum — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (AGMETEOR) [reply .travis.yml]: Yes, it's installed.

File: .travis.yml (hunk 2 above)
Comment (AGMETEOR) [reply .travis.yml:29]: Yes, it is.

File: .coveragerc (hunk 3 above)
Comment (AGMETEOR) [reply .coveragerc]: More color on this.

File: .travis.yml (hunk 1 above)
Comment (pluwum) [reply .travis.yml]: @AGMETEOR I don't see it in requirements.txt 🤔

File: .travis.yml (hunk 2 above)
Comment (pluwum) [reply .travis.yml:29]: @AGMETEOR I don't see it in requirements.txt 🤔

File: .coveragerc (hunk 3 above)
Comment (pluwum) [reply .coveragerc]: The red arrow shows that you need to add a new line at the end of the file

**Review by pluwum — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (pluwum) [reply .travis.yml:29]: The red arrow shows that you need to add a new line at the end of the file. Please add it.

**Review by pluwum — CHANGES_REQUESTED** [review]

**Comment by pluwum** [comment]
@AGMETEOR Also, please find out why the build is failing.

**Pull request merged**